src/Machete.HL7.Tests/Segments/FT.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Machete.HL7.Tests.Segments
+{
+    using System;
+    using System.Runtime.Serialization;
+
+
+    [Serializable]
+    public class FT :
```

Comment:
Was trying to use as part of a test. How the test harness is currently set up I can only load the default spec or the user-defined spec and I needed some of that core functionality in the user-defined spec